src/app.jsx

```diff
@@ -237,6 +257,33 @@ const NavigatorCardBody = ({ currentFilter, files, isGrid, setPath, path, pathIn
     }
 };
 
+const SidebarHeader = ({ selected }) => {
+    return (
+        <CardHeader>
+            <CardTitle component="h2">{selected === null ? _("Current directory") : selected.name}</CardTitle>
+        </CardHeader>
+    );
+};
+
+const SidebarBody = ({ selected }) => {
+    return (
+        <CardBody>
+            {selected === null &&
+                <ul>
```

Comment:
YOu maybe want to use https://patternfly-react-v5.surge.sh/components/description-list/ here